setup.py

```diff
@@ -19,9 +19,10 @@
 import sys
 import six
 
+exec(open('ipopt/version.py').read())
```

Comment:
Would a `from ipopt.version import __version__` work, too? Would be way less error prone.